src/overlays/actors/ovl_Boss_Ganon2/z_boss_ganon2.c

```diff
@@ -52,342 +56,3008 @@ const ActorInit Boss_Ganon2_InitVars = {
+            this->unk_3A4.y = player->actor.world.pos.y + 40.0f + 10.0f;
+            this->unk_3A4.z = player->actor.world.pos.z + 20.0f + 10.0f;
+            this->unk_3B0.x = player->actor.world.pos.x - 20.0f;
+            this->unk_3B0.y = ((player->actor.world.pos.y + 10.0f + 60.0f) - 20.0f) - 3.0f;
+            this->unk_3B0.z = (player->actor.world.pos.z - 40.0f) - 10.0f;
+            if (this->unk_398 == 0xA) {
+                temp_v0_3 = globalCtx->specialEffects;
+                temp_v0_3->unk_2E = 1;
+                temp_v0_3->position.x = sZelda->actor.world.pos.x + 50.0f + 10.0f;
+                temp_v0_3->position.y = sZelda->actor.world.pos.y + 350.0f;
+                temp_v0_3->position.z = sZelda->actor.world.pos.z - 25.0f;
+                temp_v0_3->velocity.x = 0.0f;
+                temp_v0_3->velocity.z = 0.0f;
+                temp_v0_3->velocity.y = -30.0f;
+                this->unk_39C = 0x1A;
+                this->unk_398 = 0;
+            } else {
+                break;
+            }
+        case 26:
+            this->unk_3A4.x = sZelda->actor.world.pos.x + 100.0f + 30.0f;
+            this->unk_3A4.y = sZelda->actor.world.pos.y + 10.0f;
+            this->unk_3A4.z = sZelda->actor.world.pos.z + 5.0f;
+            this->unk_3B0.x = sZelda->actor.world.pos.x;
+            this->unk_3B0.y = sZelda->actor.world.pos.y + 30.0f;
+            this->unk_3B0.z = sZelda->actor.world.pos.z - 20.0f;
+            this->unk_3BC.z = -0.5f;
+            if (this->unk_398 == 0xD) {
+                sZelda->unk_3C8 = 6;
+            }
+            if (this->unk_398 == 0x32) {
+                this->unk_39C = 0x1B;
+                this->unk_398 = 0;
+            }
+            break;
+        case 27:
+            this->unk_3BC.z = 0.0f;
+            if (this->unk_398 == 4) {
+                func_8002DF54(globalCtx, &this->actor, 0x58);
+            }
+            this->unk_3A4.x = player->actor.world.pos.x - 20.0f;
+            this->unk_3A4.y = player->actor.world.pos.y + 50.0f;
+            this->unk_3A4.z = player->actor.world.pos.z;
+            this->unk_3B0.x = player->actor.world.pos.x;
+            this->unk_3B0.y = player->actor.world.pos.y + 50.0f;
+            this->unk_3B0.z = player->actor.world.pos.z;
+            if (this->unk_398 == 0x1A) {
+                D_8090EB30 = globalCtx->actorCtx.actorLists[ACTORCAT_ITEMACTION].head;
+                while (D_8090EB30 != NULL) {
+                    if (D_8090EB30->id == ACTOR_EN_ELF) {
+                        this->unk_3A4.x = D_8090EB30->world.pos.x - 30.0f;
+                        this->unk_3A4.y = D_8090EB30->world.pos.y;
+                        this->unk_3A4.z = D_8090EB30->world.pos.z;
+                        this->unk_3B0.x = D_8090EB30->world.pos.x;
+                        this->unk_3B0.y = D_8090EB30->world.pos.y;
+                        this->unk_3B0.z = D_8090EB30->world.pos.z;
+                        break;
+                    }
```

Comment:
nothing to add here, just upset that morita didnt use the navi pointer in the player struct lol